includes/Core/Assets/Assets.php

```diff
@@ -1121,4 +1137,51 @@ protected function get_product_post_type() {
 
 		return null;
 	}
+
+	/**
+	 * Updates the plugins inline data based on plugin status.
+	 *
+	 * @param array $inline_data The inline base data to be updated.
```

Comment:
```suggestion
	 * @param array $inline_data The inline Site Kit base data.
```

Let's not say we're "updating" the data, as it sort of implies variable mutation 😅